content/spin/v3/dynamic-configuration.md

```diff
@@ -260,9 +262,25 @@ container = "<cosmos-container>"
 
 > Note: The CosmosDB container must be created with the default partition key, `/id`.
 
-Whilst a single default store may be sufficient for certain application use cases, each Spin application can be configured to support multiple stores of any `type`, as shown in the `runtime-config.toml` file below:
+### AWS DynamoDB Key Value Store Provider
+
+To use an Amazon Web Services DynamoDB database as a backend for Spin's key/value store, change the type to `aws_dynamo` and specify your database account details:
+
+```toml
+[key_value_store.default]
+type = "aws_dynamo"
+region = "<aws-region>"   # e.g. "us-east-1"
+table = "<dynamo-table>"  # e.g. "spin-key-values"
+consistent_read = true    # depending on your consistency requirement
+```
+
+You may optionally provide `access_key` and `secret_key` credentials; otherwise, Spin picks these up from your [AWS environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
```

Comment:
Possibly worth also noting the possibility of passing `token` alongside these -- it comes from creating short lived credentials for a user/role using e.g. STS